biosimulations/jest.config.js

```diff
@@ -6,4 +6,5 @@ module.exports = {
   resolver: '@nrwl/jest/plugins/resolver',
   moduleFileExtensions: ['ts', 'js', 'html'],
   coverageReporters: ['lcov'],
+  testEnvironment: 'node',
```

Comment:
This will cause issues for the Angular apps. 
You can set this options for the apps individually

Reply:
Yes, I'll remove from here.